src/stylesheets/components/_inverted-button.scss

```diff
@@ -73,16 +13,19 @@ $app-button-inverted-shadow-colour: shade($app-button-inverted-foreground-colour
   box-shadow: 0 $button-shadow-size 0 $app-button-inverted-shadow-colour;
 }
 
-.app-button--inverted:hover,
-.app-button--inverted:focus {
-  color: $app-button-inverted-foreground-colour !important;
-  background-color: tint($app-button-inverted-foreground-colour, 90%);
+.app-button--inverted:hover {
+  color: $app-button-inverted-foreground-colour;
+  background-color: $app-button-inverted-hover-background-colour;
 }
 
-.govuk-button--start.app-button--inverted {
-  background-image: url("/images/components/icon-pointer--blue.png");
+.app-button--inverted:focus:not(:hover) {
+  color: $govuk-focus-text-colour;
+  background-color: $govuk-focus-colour;
+}
 
-  @include govuk-device-pixel-ratio {
-    background-image: url("/images/components/icon-pointer--blue-2x.png");
-  }
+.app-button--inverted:active {
```

Comment:
This is close but if you are focused and hovered (not active), the styling is slightly off.

I think if you change this active selector to include focus you'll be fine.

```scss
.app-button--inverted:active 
.app-button--inverted:focus {
  // ...
}
```

Reply:
Good spot, I hadn't noticed that it had a thinner border compared to original buttons in the same state.